### ISSUE TITLE
Support for postgresql  42.7.2 - java.lang.NoSuchFieldError: checkConnectionQuery

### ISSUE THREAD

**Issue by vadimbakaev** [kamon-io/Kamon#1321]
It seems that `checkConnectionQuery` is no longer available from version 42.7.2

See the release notes: https://github.com/pgjdbc/pgjdbc/releases/tag/REL42.7.2

https://github.com/pgjdbc/pgjdbc/pull/3101

It looks like it is used in this place: https://github.com/kamon-io/Kamon/blob/25386b2598c817ee939c5d568e77389eb28a2b39/instrumentation/kamon-jdbc/src/main/scala/kamon/instrumentation/jdbc/StatementInstrumentation.scala#L192

```
java.lang.NoSuchFieldError: checkConnectionQuery
	at org.postgresql.jdbc.PgConnection.getCheckConnectionStatement(PgConnection.java)
	at kamon.instrumentation.jdbc.PgConnectionIsAliveAdvice$.enter(StatementInstrumentation.scala:199)
	at org.postgresql.jdbc.PgConnection.isValid(PgConnection.java:1520)
	at com.zaxxer.hikari.pool.PoolBase.checkValidationSupport(PoolBase.java:460)
	at com.zaxxer.hikari.pool.PoolBase.checkDriverSupport(PoolBase.java:443)
	at com.zaxxer.hikari.pool.PoolBase.setupConnection(PoolBase.java:412)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:365)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:202)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:461)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:550)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:98)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:80)
	...
```

Version 42.7.2 has security fixes and Kamon blocks the migration, could you take a look, please?

**Comment by Dichotomia** [comment]
We are encountering the same problem here.

**Comment by choffmeister** [comment]
This is the workaround for us currently, so that we can update to the fixed pgsql.

```
# reference.conf
kanela.modules.jdbc.enabled = no
```

Ofc this means, that we currently have no instrumentation for jdbc, but at least fixed the vuln and all other kamon stuff still works.

**Comment by hughsimpson** [comment]
Just ran into this myself. Will push out a fix

**Comment by hughsimpson** [comment]
Will need @ivantopo in order to release because reasons. Will leave this issue open until it's successfully out

**Comment by hughsimpson** [comment]
Resolved as of v2.7.1
